### PR TITLE
pre-fill restore form element "replace" according to selected restore job in the webui

### PR DESCRIPTION
**Pull request by fbergkemper**

### CODE REVIEW

File: core/src/lib/util.cc [hunk 1]
```diff
@@ -584,6 +584,26 @@ const char* job_type_to_str(int type)
   return str;
 }
 
+const char* job_replace_to_str(int replace)
+{
+  const char* str = NULL;
```
Comment (franku) [inline core/src/lib/util.cc]:
```suggestion
  // remove the temporary variable and return the from each case level directly
```

File: core/src/lib/util.cc [hunk 2]
```diff
@@ -584,6 +584,26 @@ const char* job_type_to_str(int type)
...
+      break;
+    case REPLACE_IFNEWER:
+      str = _("ifnewer");
+      break;
+    case REPLACE_IFOLDER:
+      str = _("ifolder");
+      break;
+    case REPLACE_NEVER:
+      str = _("never");
+      break;
```
Comment (franku) [inline core/src/lib/util.cc]:
```suggestion
      break;
    default:
      return "?? Unknown parameter";
    
```

File: core/src/lib/util.cc [hunk 3]
```diff
@@ -584,6 +584,26 @@ const char* job_type_to_str(int type)
...
+  }
+  return str;
```
Comment (franku) [inline core/src/lib/util.cc]:
```suggestion
  // see comment above
```

File: core/src/lib/util.cc (hunk 2 above)
Comment (fbergkemper) [reply core/src/lib/util.cc]: default case added

File: core/src/lib/util.cc (hunk 1 above)
Comment (fbergkemper) [reply core/src/lib/util.cc]: For consistency, I have applied the style of the existing functions to the new one.

**Comment by fbergkemper** [comment]
Thanks for the review.

**Pull request merged**